src/pages/exhibition/EnterScan.tsx

```diff
@@ -12,6 +12,25 @@ const EnterScan: React.VFC = () => {
   const aspida = useAspidaClient();
   const { currentUser, currentUserId } = useAuthState();
 
+  const [isFull, setIsFull] = useState<boolean>(false);
+
+  const checkIsFull = useCallback(async () => {
+    const status = await api(aspida)
+      .exhibitions._id(currentUser?.id || "")
+      .$get();
+    const sum = Object.entries(status.count)
+      .map(([, value]) => value)
```

Comment:
わたしのクソコードに引っ張られてたらごめん

```suggestion
    const sum = Object.values(status.count)
```